app/Http/Review/ReviewController.php

```diff
@@ -56,9 +56,9 @@ public function create()
      */
     public function store(Request $request)
     {
-        $this->reviews->create($request->all());
+        if($request) {$this->reviews->create($request->all());
 
-        return redirect('/reviews');
+        return redirect('/reviews')};
```

Comment:
The closing bracket `}` for the `if` condition is misplaced. It should be placed at the end of the `if` block, not after the `redirect` method.